example/core_tutorial/05_slice/slice_example.cpp

```diff
@@ -64,8 +64,12 @@ void sliceExample()
       allocated on NVIDIA devices use `Kokkos::CudaSpace` instead of
```

Comment:
This comment will need to be updated to describe that this example uses default spaces to work across all supported backends, but explicit choices like HostSpace and CudaSpace can also be used.